src/components/filesystem/index.js

```diff
@@ -270,15 +271,20 @@ module.exports = {
             if (this.isSecretLink)
 		return;
 	    var that = this;
-	    this.context.getSpaceUsage().thenApply(u => that.usageBytes = u.value_0);
+	    this.context.getSpaceUsage().thenApply(u => {
+	        that.usageBytes = u
```

Comment:
do we not need the value_0?

Reply:
No. We call .toString() on the Long object.

Reply:
Wasn't there a reason for it? Maybe to do with handling the 4GiB barrier?

Reply:
It wasn't consistent in any case (quotaBytes versus usageBytes) . It is now. I have tested it with large numbers.

Reply:
What sizes have you tested it with?

Reply:
Usage: 31.68 KiB / 200 GiB
Usage: 31.67 KiB / 20000 GiB
Usage: 31.67 KiB / 2000000 GiB